### PR TITLE
Play mode detection improvement

### LINKED ISSUE #1289: Despawn vehicle button missing
### Describe the problem

The red 'Despawn vehicle' button that appears on vehicle info panels is missing in 11.6.2 TEST.

I'm using build from PR #1284 if that makes any difference?

### Steps to reproduce

1. Click a vehicle to show it's info panel
2. The red despawn vehicle button is missing

### Log files

* [output_log.zip](https://github.com/CitiesSkylinesMods/TMPE/files/7885217/output_log.zip)
* [TMPE.log](https://github.com/CitiesSkylinesMods/TMPE/files/7885218/TMPE.log)

### Savegame?

n/a

### Screenshots?

image
image

### PR DESCRIPTION
**Pull request by krzychu124**
Related #1286 - one scene was not enough for all cases
Closes #1289 

[Build zip](https://ci.appveyor.com/api/projects/krzychu124/tmpe/artifacts/TMPE.zip?branch=1289-play-mode-detection-fix)

### CODE REVIEW

File: TLM/TLM/Lifecycle/TMPELifecycle.cs [hunk 1]
```diff
@@ -62,7 +62,8 @@ public class TMPELifecycle : MonoBehaviour {
         /// <summary>
         /// determines whether Game mode as oppose to edit mode (eg asset editor).
         /// </summary>
-        internal static bool PlayMode => SceneManager.GetActiveScene().name.Equals("InGame");
+        internal static bool PlayMode => SceneManager.GetActiveScene().name.Equals("Ingame") ||
```
Comment (kvakvs) [inline TLM/TLM/Lifecycle/TMPELifecycle.cs:65]: Twice accessing global.function().field, can this be expressed with a temporary variable for name?

**Review by kvakvs — APPROVED** [review]
Nice

**Review by aubergine10 — APPROVED** [review]
LGTM 👍 
image

**Review by chameleon-tbn — APPROVED** [review]
Works as expected

**Pull request merged**